seq2science/rules/configuration_generic.smk

```diff
@@ -25,14 +25,16 @@ from snakemake.utils import min_version
 from snakemake.exceptions import TerminatedException
 
 import seq2science
-from seq2science.util import samples2metadata, prep_filelock, url_is_alive, color_parser
+from seq2science.util import UniqueKeyLoader, samples2metadata, prep_filelock, url_is_alive, color_parser
 
 
 # get the cache and config dirs
 CACHE_DIR = os.path.join(xdg.XDG_CACHE_HOME, "seq2science", seq2science.__version__)
 
 # config.yaml(s)
 
+# convert all config keys to lower case (upper case = user typo)
+config =  {k.lower(): v for k, v in config.items()}
```

Comment:
Not a fan! Which one would be kept? Better to crash :fire: :see_no_evil:

Reply:
good one, will do.

Reply:
The loader checks for this now. This line is still in, in case the lowered case kay is not overlapping existing keys.